web/docs/operators/diagnostics.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_custom_props:
+  operator:
+    source: true
+---
+
+# diagnostics
+
+Retrieves diagnostic events from a Tenzir node.
+
+## Synopsis
+
+```
+diagnostics [--live]
+```
+
+## Description
+
+The `diagnostics` operator retrieves diagnostic events from a Tenzir
+node.
+
+### `--live`
+
+Work on all diagnostic events as they are generated in real-time instead of on
+diagnostic events persisted at a Tenzir node.
+
+## Schemas
+
+Tenzir emits diagnostic information with the following schema:
+
+### `tenzir.diagnostic`
+
+Contains detailed information about the diagnostic.
+
+|Field|Type|Description|
+|:-|:-|:-|
+|`pipeline_id`|`string`|The ID of the pipeline that received the diagnostic.|
```

Comment:
```suggestion
|`pipeline_id`|`string`|The ID of the pipeline that created the diagnostic.|
```

Reply:
Changed in https://github.com/tenzir/tenzir/pull/3828/commits/8e9c2cd861c547cb5fb8694fb44d02b70145f6e8